Touk_BookingApp/src/main/java/com/lc/touk_bookingapp/dto/ReservationResultDTO.java

```diff
@@ -0,0 +1,18 @@
+package com.lc.touk_bookingapp.dto;
+
+import lombok.Getter;
+import lombok.Setter;
+
+import java.time.LocalDateTime;
+
+@Getter
+@Setter
+public class ReservationResultDTO {
+    private double totalAmountToPay;
```

Comment:
czy wiesz dlaczego double to typ, który nie powinien być używany do przechowywania $$?